internal/workflow/continue.go

```diff
@@ -211,7 +211,8 @@ func (w *Session) complete(msg *dipper.Message) {
 	}
 	dipper.Logger.Infof("[workflow] session [%s] completing with msg labels %+v", w.ID, msg.Labels)
 	if w.ID != "" {
-		if _, ok := w.store.sessions[w.ID]; ok {
+		res := dipper.IDMapGet(w.store.sessions, w.ID).(SessionHandler)
+		if res != nil {
```

Comment:
This can be written in one line in go style, or just skip the assignment just compare the return value directly. I don't think we need that variable.
```go
if ....; ... {
  ...
} 
```
or
```go
if dipper.IDMapGet(...) != nil {
  ...
}
```

Reply:
ah let me get on that